src/app/window-handler.ts

```diff
@@ -75,6 +77,25 @@ export class WindowHandler {
         };
     }
 
+    /**
+     * Notification settings window opts
+     */
+    private static getNotificationSettingsOpts(): ICustomBrowserWindowConstructorOpts {
+        return {
+            width: 460,
+            height: 360,
+            show: false,
+            modal: true,
+            autoHideMenuBar: true,
+            webPreferences: {
+                sandbox: true,
+                nodeIntegration: false,
+                devTools: false,
```

Comment:
should we also have `contextIsolation` as true?

Reply:
added